Source/Blazorise/TextEdit.razor.cs

```diff
@@ -1,129 +1,141 @@
-            }
-
-            base.Dispose( disposing );
-        }
-
-        protected override Task OnInternalValueChanged( string value )
-        {
-            return TextChanged.InvokeAsync( value );
-        }
-
-        protected override Task<ParseValue<string>> ParseValueFromStringAsync( string value )
-        {
-            return Task.FromResult( new ParseValue<string>( true, value, null ) );
-        }
-
-        #endregion
-
-        #region Properties
-
-        protected string Type => Role.ToTextRoleString();
-
-        protected string Mode => InputMode.ToTextInputMode();
-
-        protected override string InternalValue { get => Text; set => Text = value; }
-
-        /// <summary>
-        /// Sets the role of the input text.
-        /// </summary>
-        [Parameter] public TextRole Role { get; set; } = TextRole.Text;
-
-        [Parameter] public TextInputMode InputMode { get; set; } = TextInputMode.None;
-
-        /// <summary>
-        /// Gets or sets the text inside the input field.
-        /// </summary>
-        [Parameter] public string Text { get; set; }
-
-        /// <summary>
-        /// Occurs after text has changed.
-        /// </summary>
-        [Parameter] public EventCallback<string> TextChanged { get; set; }
-
-        /// <summary>
-        /// Gets or sets an expression that identifies the text value.
-        /// </summary>
-        [Parameter] public Expression<Func<string>> TextExpression { get; set; }
-
-        /// <summary>
-        /// Occurs when a key is pressed down while the control has focus.
-        /// </summary>
-        [Parameter] public EventCallback<KeyboardEventArgs> KeyDown { get; set; }
-
-        /// <summary>
-        /// Occurs when a key is pressed while the control has focus.
-        /// </summary>
-        [Parameter] public EventCallback<KeyboardEventArgs> KeyPress { get; set; }
-
-        /// <summary>
-        /// Occurs when a key is released while the control has focus.
-        /// </summary>
-        [Parameter] public EventCallback<KeyboardEventArgs> KeyUp { get; set; }
-
-        /// <summary>
-        /// A string representing a edit mask expression.
-        /// </summary>
-        [Parameter] public string EditMask { get; set; }
-
-        /// <summary>
-        /// Specify the mask type used by the editor.
-        /// </summary>
-        [Parameter] public MaskType MaskType { get; set; }
-
-        /// <summary>
-        /// Specifies the maximum number of characters allowed in the input element.
-        /// </summary>
-        [Parameter] public int? MaxLength { get; set; }
-
-        /// <summary>
-        /// The size attribute specifies the visible width, in characters, of an <input> element.
-        /// </summary>
-        /// <see cref="https://www.w3schools.com/tags/att_input_size.asp"/>
-        [Parameter] public int? VisibleCharacters { get; set; }
-
-        #endregion
-    }
-}
+﻿#region Using directives

```

Comment:
I'm trying to figure why entire `TextEdit.razor.cs` file is marked as changed. You only added three properties...

Reply:
I'm developing in vscode on a mac, maybe thats it?

Reply:
I tried formatting it. Nothing changes. Maybe it switched to different file encoding?

Reply:
when I look at it in vscode it just says that I only added those 3 properties, I don't see any whitespace differences.
Encoding is UTF-8 with BOM

Reply:
The same for me, I just checked. UTF-8 with BOM.